### PR TITLE
rpi-5.4.y: add Sensirion SPS30 to i2c-sensor overlay

### PR DESCRIPTION
**Pull request by ynezz**

### CODE REVIEW

**Comment by ynezz** [comment]
Should I create patches/PRs for 5.5 and 5.6 as well, or this could be cherry-picked?

**Comment by pelwell** [comment]
This looks OK, but it needs an entry in the README (watch the formatting - no TABs, same indents).

It will be cherry-picked to other branches

**Comment by ynezz** [comment]
@pelwell Done.

**Comment by pelwell** [comment]
The address appears to be modifiable, but you don't list the legal values or default.

**Comment by ynezz** [comment]
> The address appears to be modifiable, but you don't list the legal values or default.

@pelwell AFAIK it's fixed address, added that info.

File: arch/arm/boot/dts/overlays/i2c-sensor-overlay.dts [hunk 1]
```diff
@@ -231,11 +231,25 @@
...
+				status = "okay";
+			};
+		};
+	};
 
 	__overrides__ {
 		addr =  <&bme280>,"reg:0", <&bmp280>,"reg:0", <&tmp102>,"reg:0",
 			<&lm75>,"reg:0", <&hdc100x>,"reg:0", <&sht3x>,"reg:0",
-			<&ds1621>,"reg:0", <&bme680>,"reg:0";
+			<&ds1621>,"reg:0", <&bme680>,"reg:0", <&sps30>,"reg:0";
```
Comment (pelwell) [inline arch/arm/boot/dts/overlays/i2c-sensor-overlay.dts]: If addresses are fixed, you don't need this change.

Comment (ynezz) [reply arch/arm/boot/dts/overlays/i2c-sensor-overlay.dts]: Ok, fixed.

**Pull request merged**